app/src/pages/common/settingsPage/notificationsTab/notificationsTab.jsx

```diff
@@ -46,24 +50,41 @@ export class NotificationsTab extends Component {
     updateNotificationsConfig: PropTypes.func,
     projectRole: PropTypes.string,
     userRole: PropTypes.string,
+    emailPlugin: PropTypes.object,
+    emailGlobalIntegrations: PropTypes.array,
   };
   static defaultProps = {
     enabled: false,
     cases: [],
     updateNotificationsConfig: () => {},
     projectRole: '',
     userRole: '',
+    emailPlugin: {},
+    emailGlobalIntegrations: [],
   };
 
-  isAbleToEditForm = () =>
+  isAbleToEditNotificationCaseList = () =>
     canConfigreEmailNotifications(this.props.userRole, this.props.projectRole);
 
+  isAbleToEditNotificationsEnableForm = () =>
+    canConfigreEmailNotifications(this.props.userRole, this.props.projectRole) &&
+    this.props.emailPlugin.enabled &&
```

Comment:
Please create the specific selector (f.e. `isEmailIntegrationAvailableSelector`) for this in `controllers/plugins/selectors`, you can use `availableIntegrationsByPluginNameSelector` (also you should check whether project integrations exists).